src/ontology/cl-edit.owl

```diff
@@ -29659,7 +29659,6 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25303526"^^xsd:string) r
 AnnotationAssertion(rdfs:label obo:CL_4023122 "oxytocin receptor sst GABAergic cortical interneuron")
 SubClassOf(obo:CL_4023122 obo:CL_0000099)
 SubClassOf(obo:CL_4023122 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956))
-SubClassOf(obo:CL_4023122 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0004990))
```

Comment:
based on your [explanation](https://github.com/obophenotype/cell-ontology/pull/1511#issue-1175608051), this seems reasonable